packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -13,30 +15,80 @@ import useId from '../utils/useId';
 import useIsFocusVisible from '../utils/useIsFocusVisible';
 import useControlled from '../utils/useControlled';
 import useTheme from '../styles/useTheme';
+import tooltipClasses, { getTooltipUtilityClass } from './tooltipClasses';
 
 function round(value) {
   return Math.round(value * 1e5) / 1e5;
 }
 
-function arrowGenerator() {
-  return {
-    '&[data-popper-placement*="bottom"] $arrow': {
+const overridesResolver = (props, styles) => {
+  const { styleProps } = props;
+
+  return deepmerge(styles.popper || {}, {
```

Comment:
As this component does not have a `Root` I would recommend creating two separate components `TooltipPopper` - you already have this one, and one wrapper for the `children` that will have the classes related to the `tooltip`. With that we will create two `overridesResolver` functions, once for each. At this moment I don't have better idea for solving this component. Cc @oliviertassinari 

PS: The overrides resolver just spreads these styles on the `root` element, which in this case does not exists, that's why we need two separate overrides resolvers for this component.

Reply:
> and one wrapper for the children

I don't follow the use case. From what I understand, we don't need to apply styles to the children